src/i18n/messages-types.ts

```diff
@@ -157,6 +157,9 @@ export interface I18nFormatArgTypes {
     "i18nStrings.copyButtonText": never;
   }
```

Comment:
I will confirm with that additions here are needed.  Please ingnore all changes to the i18n directory until I do